homeworks/01-bubble-sort/src/containers/BubbleSortMain/services/rng.ts

```diff
@@ -0,0 +1,22 @@
+type GenerateRandomNumber = (min: number, max: number) => number;
+
+export const generateRandomInteger: GenerateRandomNumber = (min: number, max: number): number => {
```

Comment:
Same

If you define a type for the whole function, then you don't need to define a type for arguments (return type, and whatever else), because it's inferred automatically.